Tests/MSFT_xIPAddress.Tests.ps1

```diff
@@ -105,7 +127,7 @@ InModuleScope MSFT_xIPAddress {
 
             It 'should be $true' {
                 $Splat = @{
-                    IPAddres = '192.168.0.1'
+                    IPAddress = '192.168.0.1'
                     InterfaceAlias = 'Ethernet'
                 }
                 $Result = ValidateProperties @Splat
```

Comment:
[Assert-VerifiableMocks](https://github.com/pester/Pester/wiki/Assert-VerifiableMocks) and [Assert-MockCalled](https://github.com/pester/Pester/wiki/Assert-MockCalled) are the commands.

You can also look at [xWebsite's tests](https://github.com/PowerShell/xWebAdministration/blob/dev/Tests/MSFT_xWebsite.Tests.ps1) as an example of how to do some Pester testing specific to DSC.